### PR TITLE
cirrus: Build with FreeBSD 12.2 in CirrusCI

### PR DESCRIPTION
**Pull request by emilengler**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ task:
     matrix:
       # A stable 13.0 image likely won't be available before early 2021
       # image_family: freebsd-13-0-snap
+      image_family: freebsd-12-2
       image_family: freebsd-12-1
```
Comment (jay) [inline .cirrus.yml:32]: can we just use the latest 12x instead of both, is there a disadvantage to that

**Comment by dfandrich** [comment]
> can we just use the latest 12x instead of both, is there a disadvantage to that

That's what I'd do, too.

File: .cirrus.yml (hunk 1 above)
Comment (emilengler) [reply .cirrus.yml:32]: I would ask the opposite. Is there a good reason to not to? Many people are still using 12.1 and AFAIK it's not dead. See freebsd.org, it is still listed on Downloads there

Comment (bagder) [reply .cirrus.yml:32]: I suppose the question is if the difference between the two versions 12.1 and 12.2 is big enough to warrant two different jobs. We don't run different jobs on other systems for different release minor version changes.

Comment (emilengler) [reply .cirrus.yml:32]: After investigating the changelog I came to the conclusion that it's probably better to keep both versions. For example make(1) was updated which is relevant for our build system.

**Review by bagder — APPROVED** [review]
Okay, let's go with dual 12 for a while and see if we learn anything and we can decide later if and when we should disable the 12.1 build

**Pull request closed without merging**